hail/src/main/scala/is/hail/expr/ir/agg/Extract.scala

```diff
@@ -42,10 +41,7 @@ case class Aggs(postAggIR: IR, init: IR, seqPerElt: IR, aggs: Array[PhysicalAggS
   def serializeSet(i: Int, i2: Int, spec: BufferSpec): IR =
     SerializeAggs(i * nAggs, i2, spec, aggs)
 
-  def eltOp(ctx: ExecuteContext, optimize: Boolean = true): IR = if (optimize)
-    Optimize(seqPerElt, true, "eltOp", ctx)
-  else
-    seqPerElt
+  def eltOp(ctx: ExecuteContext): IR = seqPerElt
```

Comment:
why did optimize go away?

Reply:
It was redundant. This IR is going into `Compile`, which optimizes itself.